src/codec.ts

```diff
@@ -58,8 +58,9 @@ type DateFields = [number, number, number];
  * @extends Date
  *
  * @param {string|number} [date] String representing the date or number
- *     representing the year.
- * @param {number} [month] Number representing the month.
+ *     representing the year. If year is a number between 0 and 99, then year is
+ *     is assumed to be 1900 + year.
```

Comment:
`If the year` ... `then the year is`
an extra `is` on line 62

Reply:
Fixed.